sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/AbstractRulesDefinitionTest.java

```diff
@@ -37,6 +37,7 @@ public class AbstractRulesDefinitionTest {
 
   private static final String PCI_DSS_RULE_KEY = "S1115";
```

Comment:
CAYC: this rule is deleted, and it's not C#. Maybe we can replace it with another one?

Reply:
I also stumbled upon this at first, but this is not actually S1115. The rule IDs here refer to some fake rules in [AbstractRulesDefinitionTest](https://github.com/SonarSource/sonar-dotnet/tree/87aff2914deef3bb95faafa0cd0d0e93b268937e/sonar-dotnet-shared-library/src/test/resources/AbstractRulesDefinitionTest).